src/main/resources/db/migration/V1_44__added_version_to_appearance_audit.sql

```diff
@@ -0,0 +1,43 @@
+ALTER TABLE juror_mod.appearance_audit
```

Comment:
Please get second opinion/review on database changes

Reply:
I will check with Craig :)

Reply:
I'm happy with the SQL changes 👍